scripts/sort_contibutors/main.py

```diff
@@ -0,0 +1,69 @@
+import requests
+import json
+import os
+from dotenv import load_dotenv
+
+# It's just for debugging purpose
+def print_list(user_list : list, label : str):
+	print(label)
+	for element in user_list:
+		print(element['username'],element['ranking'])
+	print("=============================================")
+
+# This function parses the contribution list, sorting
+# the users per its ranks
+def parse_contributor_list(user_list : list) -> list:
+	ret = []
+	top_contributors = []
+	contributors = []
+
+	for element in user_list:
+		ranking = element['contributions']
+		username = element['login']
+
+		# Filter the github bots
+		if '[bot]' not in username:
+			if ranking >= 100:
+					top_contributors.append({'username':username,'ranking':ranking})
+			else:
+				contributors.append({'username':username,'ranking':ranking})
+
+	ret.append(top_contributors)
+	ret.append(contributors)
+	return ret
+
+# Retrieves the list of contributors of a repository in JSON format
+def get_contibutor_list(project : str,user_token: str) -> list:
+	headers = {'X-GitHub-Api-Version':'2022-11-28','Accept':'application/vnd.github+json','Authorization':'Bearer ' + user_token}
+	r = requests.get('https://api.github.com/repos/OWASP/'+project+'/contributors',headers=headers)
+	return parse_contributor_list(r.json())
+
+
+# =============================================================
+# THE MAIN PROGRAM STARTS HERE
+# =============================================================
+
+# Loads the .env file
+load_dotenv()
+token = os.getenv('USER_TOKEN')
+
+if token != None:
+
+	# Prints the Wrong Secrets contributors list
+	wrongsecrets_list = get_contibutor_list('wrongsecrets',token)
+	print_list(wrongsecrets_list[0],"Wrong Secrets top contributors:")
+	print_list(wrongsecrets_list[1],"\nContributors:")
```

Comment:
instead of having 3 separate lists: i want it to be condensed to 1 list of which for every user his contributions are summed up over the 3 repositories :) .

Reply:
Got it.

Reply:
It's done, I'm working on print to a file.